Reference/Searching/Examine/transforming-index-values.md

```diff
@@ -0,0 +1,120 @@
+---
+versionFrom: 8.0.0
+---
+
+# TransformingIndexValues
+
+In the [Quick Start](Quick-Start/index.md) documentation you can see how to perform a simple search with Examine. That is great if you want to search between node names or you know that you always want to search for a specific field - e.g. `bodyText`. 
+
+However, what if you want to search through several different node types and search across many different fields, you likely will not want to have a query that looks like this:
+
+```cs
+var results = searcher.CreateQuery("content").Field("nodeName", searchTerm)
+                    Or().Field("bodyText", searchTerm)
+                    Or().Field("description", searchTerm)
+                    Or().Field("about", searchTerm)
+                    Or().Field("otherText", searchTerm)
+                    .Execute();
+```
+
+This quickly becomes unmanageable, would have to add each new field to the query as you make them. 
```

Comment:
This should be

> This can be simplified

Otherwise it sounds like the APIs we are providing are not good enough out of the box and we are assuming that everyone must do it this way which isn't true.